front/pages/join.js

```diff
@@ -1,6 +1,8 @@
 import validateEmail from '../validator/emailValidator';
+import formValidator from '../validator/formValidator';
 
-export default () => {
+export default (formType) => {
+    const form = dom('[name="adherent_registration"]') || dom('[name="become_adherent"]');
```

Comment:
1) `dom('form[name="adherent_registration"]')`
2) why `|| dom(...)`? check in this JS file you work with `#adherent_registration_` fields

Reply:
there are 2 possible form in this view :s